<ui version="4.0" >
 <class>dominios</class>
 <widget class="QDialog" name="dominios" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dominios</string>
  </property>
  <widget class="QLineEdit" name="lineEditDominio" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>20</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="aceptarDominio" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
